crates/iceberg/src/spec/sort.rs

```diff
@@ -82,6 +116,14 @@ impl SortOrder {
         SortOrderBuilder::default()
     }
 
+    /// Create an unbound unsorted order
+    pub fn unsorted_order() -> SortOrder {
+        SortOrder {
+            order_id: 0,
```

Comment:
`0` is used for unsorted order id in many places, it would be better to make it a crate private constants?